apps/chat/src/components/Chat/ChatInput/ChatInputMessage.tsx

```diff
@@ -93,16 +88,16 @@ export const ChatInputMessage = ({
 
   const isError = isLastAssistantMessageEmpty || isMessageError;
 
-  const maxLength = useMemo(() => {
-    const maxLengthArray = selectedConversations.map(
-      ({ model }) =>
-        modelsMap[model.id]?.maxLength ??
-        OpenAIEntityModels[model.id]?.maxLength ??
-        defaultModelLimits.maxLength,
-    );
-
-    return Math.min(...maxLengthArray);
-  }, [modelsMap, selectedConversations]);
+  const selectedModels = useAppSelector(
+    ConversationsSelectors.selectSelectedConversationsModels,
+  );
+  const modelTokenizer =
+    selectedModels?.length === 1 ? selectedModels[0]?.tokenizer : undefined;
+  const maxTokensLength =
+    selectedModels.length === 1
+      ? selectedModels[0]?.limits?.maxRequestTokens ?? Infinity
+      : Infinity;
```

Comment:
what if two chats with different models? Maybe it makes sense to get minimum `maxRequestTokens` in this case?

Reply:
Don't think so - if they have different maxRequestTokens they can also have different tokenization alghoritms, so it will be not easy to calculate
I think we should not do complicated logic here in such case, so we rely on adapters answers